packages/simple-types/src/simple-types.ts

```diff
@@ -123,18 +125,24 @@ export function postProcess(
   templateDictionary: any,
   authentication: UserSession
 ): Promise<any> {
-  return getItemDataAsJson(itemId, authentication).then(data => {
-    if (hasUnresolvedVariables(data)) {
-      const updatedData = replaceInTemplate(data, templateDictionary);
-      // TODO: update return type on updateItemExtended
-      return updateItemExtended(
-        itemId,
-        { id: itemId },
-        updatedData,
-        authentication
-      ) as Promise<any>;
-    } else {
-      return Promise.resolve({ success: true });
-    }
-  });
+  if (type === "QuickCapture Project") {
```

Comment:
I'd consider creating and exporting another "Processor" for quick capture - basically the same way `notebook` is handled. That allows the `simpleTypes` stuff to continue to be super generic, but when we run into types that need special processing, we can isolate that off, which then makes testing simpler as well.

Reply:
Your call - we're crunched for time, but from my experience having gone through 3 iterations of this in the Hub app, delegating to type-specific handlers has always made things simpler to work with, test, extend and reason about :)